app/javascript/src/hyacinth_v1/components/shared/TopNavbar.js

```diff
@@ -98,7 +98,7 @@ function TopNavbar(props) {
         </Nav>
 
         <Nav>
-          <NavDropdown alignRight title={`${user.firstName} ${user.lastName}`}>
```

Comment:
should there be an `align="end"` here, per https://react-bootstrap.netlify.app/migrating/#dropdown

Reply:
👍